tests/istio.mk

```diff
@@ -217,3 +217,34 @@ junit-report: out_dir ${ISTIO_BIN}/go-junit-report
 # This assume istio runs in istio-system namespace, and 'skip-cleanup' was used in tests.
 dumpsys: out_dir
 	tools/dump_kubernetes.sh --output-directory ${OUT_DIR}/logs --error-if-nasty-logs
+
+# Run once for each cluster, to install helm on the cluster
+helm/init: ${HELM}
+	kubectl apply -n kube-system -f install/kubernetes/helm/helm-service-account.yaml
+	helm init --upgrade --service-account tiller
+
+# Install istio for the first time
+helm/install:
+	${HELM} install \
+	  install/kubernetes/helm/istio \
+	  --name istio-system --namespace istio-system \
+	  --set global.hub=${HUB} \
+	  --set global.tag=${TAG} \
+	  --set global.imagePullPolicy=Always \
```

Comment:
Why Always?

Reply:
For the test - yes. In general it should be the default - but because of how we test it is not (we need to fix it in 1.1).

Regarding CRDs - again, defining CRDs doesn't break a-la-carte ( sorry, not sure how to get the french accent). It's just adding definitions to k8s, no workload involved.